ms2deepscore/data_generators.py

```diff
@@ -17,22 +17,19 @@ class DataGeneratorAllSpectrums(Sequence):
     other spectrum that corresponds to a reference score as defined in same_prob_bins.
     """
     def __init__(self, spectrums_binned: List[BinnedSpectrum], spectrum_ids: list,
-                 score_array: np.ndarray, inchikey_score_mapping: np.ndarray,
-                 dim: int, **settings):
+                 labels_df: pd.DataFrame, dim: int, **settings):
```

Comment:
Could be only me, but I first stumbled a bit about the name `labels`. It of course is totally standard from the ML community side, but still might not be 100% obvious here? Maybe `reference_scores_df`? But feel free to leave it in.

Reply:
OK, I think if you stumble over it then others might as well. I renamed it to `reference_scores_df` like you suggest. :)